src/rgw/rgw_op.h

```diff
@@ -233,6 +235,11 @@ class RGWOp : public DoutPrefixProvider {
     this->store = store;
     this->s = s;
     this->dialect_handler = dialect_handler;
+    auto trace_name = std::string(name()) + " " + s->trans_id;
+    this->trace = rgw_tracer->start_trace(trace_name);
+    this->trace->set_tag("trans_id",s->trans_id);
+    this->trace->set_tag("op",name());
+    this->trace->set_baggage("trace_name", trace_name);
```

Comment:
don't think that this demonstrates good use of `set_baggage()`
we can leave it out here, and just document it in the header file